docs/detections/api/lists/lists-api-overview.asciidoc

```diff
@@ -60,13 +60,10 @@ data streams for the {kib} space (see <<lists-index-api-overview>>).
 [float]
 === Kibana role requirements
 
-To create list containers and items, the user role for the {kib} space must
-have:
+To turn on the lists feature, a user with the `manage` cluster privilege must visit the Rules page in your Kibana space. 
```

Comment:
without .lists,.items privileges feature won't be 'turn on', just `.manage` cluster is not enough

Reply:
Maybe we can rephrase it as:

To turn on the lists feature, user with the following privileges must visit the Rules page in your Kibana space:

- `manage` cluster privilege
- `read` and `write` index privileges for the `.lists` and `.items` data streams (the system data stream used for storing exception lists).
- {kib} space `All` privileges for the `Security` and `Saved Objects Management`
features (see {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges]).

Reply:
OK, let me try to summarize what I've gathered so far. It appears there's two use cases that need to be doc'd.

**Use case 1:** Create the `.lists` data stream, which can contain lists and lists items. 

To create the `.lists` data stream, a user with the following privs must visit the Rules page in your Kibana space:

- `manage` cluster privilege
- `read` and `write` index privileges for the `.lists` and `.items` data streams (the system data stream used for storing exception lists).
- {kib} space `All` privileges for the Security and Saved Objects Management features (see {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges]).

**Use case 2:** Manage lists 

To create, edit, and delete lists, a user needs the following privs:

- `read` and `write` index privileges for the `.lists` and `.items` data streams (the system data stream used for storing exception lists).
- {kib} space `All` privileges for the Security and Saved Objects Management features (see {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges]).

Is this correct? Also, do users need different privileges to manage list items? Or are the required privs the same as what they'd need to manage lists?

Reply:
Gotcha and thanks @vitaliidm for providing an example to illustrate the relationship between the  `.lists` and `.items` data streams and the objects they contain. Your explanation and examples will actually help address some of the asks in two other open doc issues (https://github.com/elastic/security-docs/issues/3754 and https://github.com/elastic/security-docs/issues/4116), so they're providing value past this one doc task. 

Moving onto summarizing this thread, here's my understanding of what we've discussed so far:
- Creating the `.lists` and `.items` data streams are pre-requisites to creating and managing lists. To create these data streams, a user with the following privs needs to visit the Rules page:
   - `manage` cluster privilege
   - `read` and `write` index privileges for the `.lists` and `.items` data streams 
   - Kibana space `All` privileges for the Security and Saved Objects Management features 
- Once the data streams have been created, users only need the following privs for managing (create/read/edit/delete) **lists** and **list items**:
   - `read` and `write` index privileges for the `.lists` and `.items` data streams 
   - Kibana space `All` privileges for the Security and Saved Objects Management features 
- Value lists are comprised of two parts: lists and list items. Additional notes on the design:
   - Lists are indexed in the `.lists` data stream and list items are indexed in the `.items` data stream. 
   - List items are indexed as separate documents, and each list item references the list that it's a part of. 

If the above is correct, I'll be make the following changes to this PR:
- [Lists API | Kibana role requirements](https://security-docs_bk_4524.docs-preview.app.elstc.co/guide/en/security/master/lists-api-overview.html#_kibana_role_requirements_3) - Will be making two changes:
    - Explain that the `.lists` and `.items` data streams are pre-requisites to creating and managing lists and describe the required privs to do so. 
    - Will also be changing [the existing text](https://github.com/elastic/security-docs/pull/4524/files#diff-42a39c514ebfe335f624a2e2af4c5bdc43cf4ab5266cf35a597763c0e76619d1R65) from "to create list containers and items..." to "to create and manage list contains and items..."
- [Detections prerequisites and requirements | Enable and access detections](https://security-docs_bk_4524.docs-preview.app.elstc.co/guide/en/security/master/detections-permissions-section.html#enable-detections-ui) - Making a couple changes to the rows I added:
   - Will change [this description](https://github.com/elastic/security-docs/pull/4524/files#diff-7071d3f8f16c85ba42ead73e4dfde38998099dc9cba07ef323600d4da086ff20R129) to "Create data streams for value lists" or "Create data streams for lists". Let me know if these work or if you think we need to go in a different direction. 
   - Will update the required privs for creating the `.lists` and `.items` data streams.

Reply:
I think `view_index_metadata` is missing in list of privs for `.lists` and `.items`.
 This privilege mentioned on https://www.elastic.co/guide/en/security/current/detections-permissions-section.html#enable-detections-ui

Reply:
Hey, @vitaliidm. [Here](https://docs.google.com/spreadsheets/d/1COlkql3q4Py4NVraBMXo4MRiljD2W-17mADfemLAV-0/edit?usp=sharing) are my testing notes. I was able to get in a lot of good testing from the UI, but was running into 403 errors when testing the list APIs. 

The TLDR of it is: 
- Users need the following privs to create the `.lists` and `.items` data streams:        
   - `manage` cluster privilege
   - `read`, `write`, `view_index_metadata`, and `manage` index privileges for the `.lists` and `.items` data streams 
   - Kibana space `All` privileges for the Security and Saved Objects Management features 
- Once the data streams are created, users need everything above, except for the `manage` cluster priv.

Reply:
Quick update on the API testing: 
I was able to figure out why I was why I was getting the 403 errors. I was testing the lists feature in a secondary Kibana space (i.e., a non-default Kibana space) and needed to specify the space name in the request URL.